src/Fixer/LanguageConstruct/FunctionToConstantFixer.php

```diff
@@ -142,32 +148,38 @@ protected function createConfigurationDefinition()
                 ->setAllowedValues([
                     (new FixerOptionValidatorGenerator())->allowedValueIsSubsetOf($functionNames),
                 ])
-                ->setDefault($functionNames)
+                ->setDefault([
+                    'get_class',
+                    'php_sapi_name',
+                    'phpversion',
+                    'pi',
```

Comment:
shouldn't we add `get_called_class` to local config and TODO for v3 to add it by default ?